packages/types/src/utils/typeguards.ts

```diff
@@ -66,3 +67,7 @@ export function isSignedBlockContents<F extends ForkBlobs>(
 ): data is SignedBlockContents<F> {
   return (data as SignedBlockContents<F>).kzgProofs !== undefined;
 }
+
+export function isElectraAttestation<F extends ForkElectra>(attestation: Attestation): attestation is Attestation<F> {
```

Comment:
```suggestion
export function isElectraAttestation(attestation: Attestation): attestation is Attestation<ForkPostElectra> {
```

I'd be interested in seeing where we use this, we maybe should be using fork / slot to determine which type it is.

Reply:
Hmmm need more thought on determining electra attestation based on slot vs presence of committee bits. 

If we receive a conflicting attestation eg. pre-electra slot with committee bits, post-electra slot with no committee bits, we need a way to handle it properly.